.github/CONTRIBUTING.md

```diff
@@ -43,18 +43,36 @@ You can (and should) run our test suite using [*tox*].
 However, you’ll probably want a more traditional environment as well.
 We highly recommend to develop using the latest Python release because we try to take advantage of modern features whenever possible.
 
-Clone the *attrs* repository to your computer:
+First [fork](https://github.com/python-attrs/attrs/fork) the repository on GitHub.
+
+Clone the fork to your computer:
 
 ```console
-$ git clone git@github.com:python-attrs/attrs.git
+$ git clone git@github.com:<your-username>/attrs.git
 ```
 
 Or if you prefer to use Git via HTTPS:
 
 ```console
-$ git clone https://github.com/python-attrs/attrs.git
+$ git clone https://github.com/<your-username>/attrs.git
+```
+
+Then add the *attrs* repository as *upstream* remote:
+
+```console
+$ git remote add upstream git@github.com:python-attrs/attrs.git
+```
+
+You can of course also rename the *origin* remote (your fork) to *fork* and add the repository as *origin*.
+
+The next step is to sync the upstream repository with your local copy:
+
+```console
+$ git fetch upstream
 ```
 
+This is important to obtain eventually missing tags, which are needed to install the development version later on. See [#1104](https://github.com/python-attrs/attrs/issues/1104) for more information.
```

Comment:
This is a bit inaccurate, though. It fetches both branches and tags. But not necessarily all.
Maybe, it makes sense to add `--tags --prune-tags`. Maybe, it makes sense to specify the branch name too.
`git fetch upstream main --tags --prune-tags` would probably represent what we want here.

Reply:
With the comment in another thread above, it could probably be just `git fetch upstream --prune-tags`.

Reply:
This will remove any local tags that do no longer exist in the remote; I guess it's useful.
I think the `--prune` option needs to be passed too for this to succeed? See [git fetch documentation](https://git-scm.com/docs/git-fetch).

Reply:
yeah that seems too specific for a "get up and contribute real quick 101" for a project that never had to remove a tag. I really want to be helpful, but if we make it too complicated, people won't read / understand it. It's a tedious balance act. :-/